terraform/modules/eks/main.tf

```diff
@@ -31,19 +48,18 @@ module "eks" {
 
   map_roles = [
     {
-      rolearn  = "arn:aws:iam::${var.iam_account_id}:role/<% .Name %>-kubernetes-admin"
+      rolearn  = "arn:aws:iam::${var.iam_account_id}:role/<% .Name %>-kubernetes-admin-${var.environment}"
       username = "<% .Name %>-kubernetes-admin"
       groups   = ["system:masters"]
     },
   ]
   cluster_iam_role_name = "k8s-${var.cluster_name}-cluster"
   workers_role_name = "k8s-${var.cluster_name}-workers"
 
-  # TODO, determine if this should be true/false
-  manage_aws_auth = true
+  # Unfortunately fluentd doesn't yet support oidc auth so we need to grant it to the worker nodes
+  workers_additional_policies = ["arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"]
```

Comment:
This was a bit of a pain. Hopefully the fluentd cloudwatch plugin is updated soon to support it.